### DISCUSSION TITLE
Bring back the old feed please

### DISCUSSION THREAD

**Discussion by DosWorld** [community/community#68087] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

From my point of view, feed have too rich javascript and cross-site scripting (and data i/o). It is conflict with my browser extensions (which keep my privacy and block adv). So, last 6 month i cant see feed. I dont have a time to spent for feed deep investigation to fix it.

**Comment by tiagoangelos** [comment]
try disabling extensions automatically for specific sites, or use the desktop version of gitHub, as a last resort try another browser. Hope this helps!

**Comment by vbarbar** [comment]
From my perspective, the feed contains a significant amount of complex JavaScript, cross-site scripting, and heavy data input/output. Unfortunately, this conflicts with my browser extensions, which are designed to enhance privacy and block advertisements. Consequently, I haven't been able to view the feed for the past six months. Regrettably, I don't currently have the time to invest in a thorough investigation to resolve this issue.

**Comment by MadTinker** [comment]
This is serious. We need to at least have the option to get rid of this new feed. It has taken a major part of github away.
